src/scriptworker/constants.py

```diff
@@ -369,10 +369,13 @@
                             "project:mobile:fennec-profile-manager:releng:signing:cert:fennec-production-signing": "fennec-profile-manager-repo",
                             "project:mobile:firefox-tv:releng:googleplay:product:firefox-tv": "firefox-tv-repo",
                             "project:mobile:firefox-tv:releng:signing:cert:production-signing": "firefox-tv-repo",
-                            "project:mobile:focus:googleplay:product:focus": "focus-repo",
-                            "project:mobile:focus:releng:signing:cert:release-signing": "focus-repo",
                             "project:mobile:reference-browser:releng:googleplay:product:reference-browser": "reference-browser-repo",
                             "project:mobile:reference-browser:releng:signing:cert:release-signing": "reference-browser-repo",
+                            "project:mobile:focus-android:releng:github:project:focus-android": "focus-repo",
+                            "project:mobile:focus-android:releng:googleplay:product:focus-android": "focus-repo",
+                            "project:mobile:focus-android:releng:signing:format:autograph_focus": "focus-repo",
+                            # beta and nightly are signed with same key as production
+                            "project:mobile:focus-android:releng:signing:cert:production-signing": "focus-repo",
```

Comment:
It looks like we also have the `release-signing` scope enabled in signingscript [here](https://github.com/mozilla-releng/scriptworker-scripts/blob/d60deec5a61255f6bd21282b2a694e26b350f3ae/signingscript/docker.d/passwords.yml#L270-L289). So while the `production-signing` scope will be restricted after this rolls out, someone could still sign with a valid key with the unrestricted `release-signing` scope.

Let's:
- determine which old branches we still support, and which signing cert scopes we still support,
- remove the cert scopes we aren't using from ci-config and signingscript, and
- make sure that any remaining non-dep cert scopes we still support are listed here.

I suspect that means we can remove `focus-android:...:release-signing` from signingscript, and that `focus:...:release-signing` is removable now or soonish, and the above restricted scope is sufficient for signing cert scopes.

I'm hoping all of my comments here are clear; let me know if you have questions?

Reply:
As I understand it, anything pre-taskgraph used the `focus` scopes, not the new `focus-android` scopes so I'm confident those are safe to remove.